chessboard/board.php

```diff
@@ -9,8 +9,25 @@
 require_once '/Users/nal64753/desktop/workspace/php-project/vendor/autoload.php';
```

Comment:
small nitpick: these should be relative paths, so if someone else downloads your code they don't have to change the path to get it to work.